### PR TITLE
Add -t <path/script> option to execute script after every check

### PR DESCRIPTION
**Pull request by adizero**
Also change the EXEC event lines color to cyan to better visually distinguish them from the DOWN/RECONNECTED event lines.

### CODE REVIEW

File: netcheck.sh [hunk 1]
```diff
@@ -107,7 +109,15 @@ PRINT_RECONNECTED() {
...
-    echo -e $COLOR_RED"$STRING_1 EXEC $COMMAND"$COLOR_RESET
+    echo -e $COLOR_CYAN"$STRING_1 EXEC $COMMAND"$COLOR_RESET
+    eval "$COMMAND"
+  fi
+}
+
+CHECK_EVENT_HOOK() {
+  if [[ $VAR_ACT_ON_CHECK = true ]]; then :
+    COMMAND="$VAR_CHECK_SCRIPT $1 &"
+    # echo -e $COLOR_CYAN"$STRING_6 EXEC $COMMAND"$COLOR_RESET
```
Comment (TristanBrotherton) [inline netcheck.sh]: Can you remove this comment if its not being used.

File: netcheck.sh [hunk 2]
```diff
@@ -94,7 +96,7 @@ PRINT_DISCONNECTED() {
 DISCONNECTED_EVENT_HOOK() {
   if [[ $VAR_ACT_ON_DISCONNECT = true ]]; then :
     COMMAND="$VAR_DISCONNECT_SCRIPT &"
-    echo -e $COLOR_RED"$STRING_2 EXEC $COMMAND"$COLOR_RESET
+    echo -e $COLOR_CYAN"$STRING_2 EXEC $COMMAND"$COLOR_RESET
```
Comment (TristanBrotherton) [inline netcheck.sh:98]: I think using red here is an appropriate color to indicate a down link and provides the maximum contrast to the nominal state of green. But I'm open to discussion if you have a strong opinions. I'd love to better understand your thought behind the change.

Comment (adizero) [reply netcheck.sh:98]: Maybe for the disconnected event red makes sense (I would not care), but I was confused about the reconnect event hook exec being in red when scrolling the console output. At first I considered changing the reconnect exec line to green, but at the end decided to change both exec events to a different color to have a single green line for reconnect event and a single red line for disconnect event when looking at the output. My reasoning is that executing an external script is neither a positive nor a negative event.

Comment (TristanBrotherton) [reply netcheck.sh:98]: Thanks for the explanation.  I would vote we leave disconnected as red (negative event), reconnect green (positive), to mirror the functionality when not executing a script to maintain familiarity to the user, but if you'd like you could break out the execution string to be a different (or no) color.

Comment (adizero) [reply netcheck.sh:98]: That is exactly what I did (at least what I have tried to do :-)). I have kept the colors for the disconnect/reconnect events and used separate color (cyan) only for the external script exec lines.

Comment (TristanBrotherton) [reply netcheck.sh:98]: In your changes you are still including $STRING_2 (LINK DOWN:) as cyan, but honestly I think this is not a big deal, lets just close this out! I really appreciate your contributions to this project. I hope its being useful for you.

**Pull request merged**